utils/README.md

```diff
@@ -0,0 +1,8 @@
+# Utilities
+
+This directory contains a number of scripts to help with development.
+
+## YML to Image of map
+
+The `yml_to_map.py` file creates a diagrammatic representation of the map based
+on the underlying yml file.
```

Comment:
Can you add a few instructions of how to use this please. Perhaps something like:

````suggestion
on the underlying yml file. Run this using:

```
python utils/yml_to_map.py src/Rooms/hub.yml
```
````